samples/react-native/src/Screens/HeavyNavigationScreen.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { Button, View, StyleSheet, Text, ScrollView } from 'react-native';
+
+import * as Sentry from '@sentry/react-native';
+import { StackNavigationProp } from '@react-navigation/stack';
+
+interface Props {
+  navigation: StackNavigationProp<any, 'HeavyNatigavionScreen'>;
+  route?: {
+    params?: {
+      manualTrack: boolean;
+    };
+  };
+}
+const buttonTitles = Array.from(
+  { length: 500 },
+  (_, index) => `Sample button ${index + 1}`,
+);
+
+const beforeEffectSpanStart = Date.now();
```

Comment:
This will be only set once when the JS bundle start executing. I think this should be in the component function body, but then its duplicate of the mount span.

Reply:
To not extend the PR review for too long I have opted to remove the `before effect` span